app/controllers/schools/partnerships_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Schools::PartnershipsController < Schools::BaseController
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
+
+  def index
+    @school = current_user.induction_coordinator_profile.schools.first
+    @partnership = @school.partnerships&.find_by(cohort: cohort)
```

Comment:
Can `school.partnerships` be falsy by itself, without `school` being falsy?

Reply:
No, not in this context at least. school.partnerships will just be a scope and find_by(cohort: cohort) would be appended to the relation before it gets evaluated.

Reply:
Then can we remove the null safety from `find_by` call on partnerships?